tasks/create-snapshot-task.yaml

```
@@ -0,0 +1,156 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-snapshot
+  namespace: rh-acs-tenant
+spec:
+  description: Creates a snapshot based on the component names, image refs and Git information passed to the task.
+  params:
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the application source code.
+    type: string
+  - name: COMPONENTS
+    description: A stringified JSON array with information about components with name, container image ref, git repo and git revision for each.
+    type: string
+  - name: PRODUCT_VERSION
+    description: Product version to record as part of the Snapshot's name to be able to identify snapshots more easily. Pass here the result from determine-image-tag for the main image.
+    type: string
+  results:
+  - name: SNAPSHOT_NAME
+    description: Name of the snapshot created by this task.
+  volumes:
+  - name: workdir
+    emptyDir: { }
+  stepTemplate:
+    volumeMounts:
+    - mountPath: /var/workdir
+      name: workdir
+    workingDir: /var/workdir
+  steps:
+  - name: render-snapshot
+    image: registry.access.redhat.com/ubi9:latest@sha256:1057dab827c782abcfb9bda0c3900c0966b5066e671d54976a7bcb3a2d1a5e53
+    env:
+    - name: APPLICATION
```

I feel that `APPLICATION_NAME` might be better than `APPLICATION`. Would you be ok to rename?
```suggestion
    - name: APPLICATION_NAME
```